internal/proxy/ui_requests.go

```diff
@@ -4,13 +4,14 @@ import (
 	"encoding/json"
 	"errors"
 	"fmt"
-	"github.com/fabric8-services/fabric8-jenkins-proxy/internal/clients"
-	"github.com/satori/go.uuid"
-	log "github.com/sirupsen/logrus"
 	"net/http"
 	"net/url"
 	"strings"
 	"time"
+
```

Comment:
does go fmt agree with this?

Reply:
That is done by go fmt. I didnt touch it.